.github/workflows/ci.yml

```diff
@@ -0,0 +1,81 @@
+name: CI Linux
+
+on:
+  push:
+    tags:
+      - '*'
+  # TODO: Remove this before merging
+  pull_request:
+  workflow_dispatch:
+    # Allow to run manually
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+          - fedora:40
+          - fedora:41
+          - fedora:42
+    container:
+      image: ${{ matrix.container }}
+
+    steps:
+      - name: Checkout code
+        # cannot use v4 yet because of https://github.com/actions/checkout/issues/1487
+        uses: actions/checkout@v3
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v6.0.1
+
+      # We cannot use the setup python action because it doesn't support all containers
+      # https://github.com/actions/setup-python/issues/527
+      - name: Set up Python
+        run: |
+          uv python install
+          uv venv
+          . .venv/bin/activate
+          echo PATH=$PATH >> $GITHUB_ENV
+
+      - name: Install dependencies
+        run: |
+          # Install dnf5
```

Comment:
I don't think this is necessary, as far as I can tell since Fedora 41 `dnf` is symlinked to `dnf5`, and Fedora 40 is supposed to be EOL in a couple weeks. Can we simplify this script by just using `dnf` everywhere?

Reply:
Thanks for the suggestion. I'll remove fedora 40 then and use `dnf` everywhere.